### ISSUE TITLE
Extra checks

### ISSUE THREAD

**Issue by smolskyaleksey** [VoIPGRID/VialerSIPLib#43]
Pls , delete extra checks in VSLCall

```
 if (error != NULL) {
            error = [NSError VSLUnderlyingError:nil
                        localizedDescriptionKey:NSLocalizedString(@"Could not setup call", nil)
                    localizedFailureReasonError:[NSString stringWithFormat:NSLocalizedString(@"PJSIP status code: %d", nil), status]
                                    errorDomain:VSLCallErrorDomain
                                      errorCode:VSLCallErrorCannotCreateCall];
        }
```
This line 
https://github.com/VoIPGRID/VialerSIPLib/blob/develop/Pod/Classes/VSLCall.m#L227
 doesn't  work correctly. We should create error every time when status != PJ_SUCCESS

**Comment by bobvoorneveld** [comment]
I think you're right, plz file a PR and we'll test...

**Comment by smolskyaleksey** [comment]
Sorry I can not do it right now. My master brunch is in an invalid state.

**Comment by bobvoorneveld** [comment]
I'll check it tomorrow and try to fix it.

**Comment by smolskyaleksey** [comment]
Thx

**Comment by bobvoorneveld** [comment]
Fixed in #49.
